static/app/views/releases/list/releaseCard/index.tsx

```diff
@@ -56,161 +56,154 @@ type Props = {
   showReleaseAdoptionStages: boolean;
 };
 
-class ReleaseCard extends Component<Props> {
-  shouldComponentUpdate(nextProps: Props) {
-    // we don't want project health rows to reorder/jump while the whole card is loading
-    if (this.props.reloading && nextProps.reloading) {
-      return false;
-    }
-
-    return true;
-  }
-
-  render() {
-    const {
-      release,
-      organization,
-      activeDisplay,
-      location,
-      reloading,
-      selection,
-      showHealthPlaceholders,
-      isTopRelease,
-      getHealthData,
-      showReleaseAdoptionStages,
-    } = this.props;
-    const {version, commitCount, lastDeploy, dateCreated, versionInfo} = release;
-
+// TODO: change to functional component
+function ReleaseCard({
+  release,
+  organization,
+  activeDisplay,
+  location,
+  reloading,
```

Comment:
how is reloading handled when the component is a functional component?

Reply:
since it's passed as a prop, it'll be handled the exact same way as it was in the class component

Reply:
Just now - we don't need to utilize it as a hack during `shouldComponentUpdate`
instead - the updates are memoized - making this much cleaner